cmd/bridge/main.go

```diff
@@ -291,6 +291,13 @@ func main() {
 		}
 	}
 
+	if len(telemetryFlags) > 0 {
+		klog.Infoln("Console telemetry flags:")
```

Comment:
There is only single telemetry flag, which contains multiple variables
```suggestion
		klog.Infoln("Console telemetry variables:")
```